pkg/storaged/content-views.jsx

```diff
@@ -440,7 +440,7 @@ function append_row(client, rows, level, key, name, desc, tabs, job_object) {
     }
 
     rows.push({
-        props: { key },
```

Comment:
This table would ideally be a https://www.patternfly.org/v4/components/table/react/tree-table/ but it's a lot of work, I am happy with this situation for the time being.

Reply:
A tree table will show/hide the children of a node, while we currently show/hide the details of a row. So as far as I understand, using a tree table would be a big change in behavior. Correct?

There are not many levels here, so showing/hiding children doesn't seem necessary to me.

Reply:
Indeed. I misthought the functionality here.